tuxemon/core/tools.py

```diff
@@ -385,7 +440,7 @@ def calc_dialog_rect(screen_rect):
     return rect
 
 
-def open_dialog(game, text, menu=None):
+def open_dialog(game, text, avatar=None, menu=None):
```

Comment:
Add a docstring for avatar and menu

Reply:
I'm not familiar with docstring and couldn't find that in the code. Can you point me to where that is please?

Reply:
Oh... they're the comments at the beginning of functions! Fixed.